fk/secretsend.go

```diff
@@ -64,29 +67,28 @@ https://download.fluidkeys.com#` + recipientEmail + `
 		return 1
 	}
 
-	_, err = encryptSecret("dummy data to test encryption", pgpKey)
+	_, err = encryptSecret("dummy data to test encryption", "", pgpKey)
 	if err != nil {
 		printFailed("Couldn't encrypt to the key:")
 		out.Print("Error: " + err.Error() + "\n")
 		return 1
 	}
 
-	out.Print("\n")
-	out.Print(colour.Info(femaleSpyEmoji + "  Type or paste your message, ending by typing Ctrl-D\n"))
-	out.Print(colour.Info("   It will be end-to-end encrypted so no-one else can read it\n\n"))
-
-	secret, err := scanUntilEOF()
-	if err != nil {
-		log.Panic(err)
-		return 1
+	var secret *string
+	var basename string
+	if filename != "" {
+		secret, err = getSecretFromFile(filename)
```

Comment:
Is this a tab / space issue? I'll look into it!

Reply:
It seems this is *fixing* the code by making this a tab. One of us needs to get our editor to use `gofmt` ... and I think it's me...

Reply:
:-/